bin/sozo/src/commands/register.rs

```diff
@@ -44,23 +47,27 @@ pub enum RegisterCommand {
 
 impl RegisterArgs {
     pub fn run(self, config: &Config) -> Result<()> {
+        trace!(target: LOG_TARGET, command=?self.command, "Executing command.");
         let env_metadata = utils::load_metadata_from_config(config)?;
 
         let (starknet, world, account, transaction, models) = match self.command {
             RegisterCommand::Model { starknet, world, account, transaction, models } => {
+                trace!(target: LOG_TARGET, models=?models, "Registering models.");
                 (starknet, world, account, transaction, models)
             }
         };
 
         let world_address = world.world_address.unwrap_or_default();
+        trace!(target: LOG_TARGET, ?world_address, "Using world address");
 
         config.tokio_handle().block_on(async {
             let world =
                 utils::world_from_env_metadata(world, account, starknet, &env_metadata).await?;
             let provider = world.account.provider();
             let world_reader = WorldContractReader::new(world_address, &provider)
                 .with_block(BlockId::Tag(BlockTag::Pending));
-
+            trace!(target: LOG_TARGET, "WorldContractReader initialized with block tag Pending.");
```

Comment:
We could print the world address here, instead of having a single print for it above.